src/ontology/pato-edit.obo

```diff
@@ -8984,8 +8984,8 @@ is_a: PATO:0002505 ! nucleated
 
 [Term]
 id: PATO:0001408
-name: ciliatedness
-def: "A cellular quality inhering in a bearer by virtue of the bearer's having thin, tail-like projections extending outwards from the cell body." [Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]
+name: ciliated
+def: "A cellular quality inhering in a bearer by virtue of the bearer's having one or more cilia as a part." [PMID:36924208, Wikipedia:http\://en.wikipedia.org/wiki/Ciliated]
```

Comment:
There is no wikipedia page for ["Ciliated".](https://en.wikipedia.org/wiki/Http:/en.wikipedia.org/wiki/Ciliated) However, there is a <https://en.wikipedia.org/wiki/Cilium> page.

Reply:
I have changed the reference to https://en.wikipedia.org/wiki/Cilium